dali/base/dafdesc.cpp

```diff
@@ -1323,6 +1324,9 @@ class CFileDescriptor:  public CFileDescriptorBase, implements ISuperFileDescrip
         attr.setown(createPTree(mb));
         if (!attr)
             attr.setown(createPTree("Attr")); // doubt can happen
+        history.setown(createPTree(mb));
```

Comment:
Isn't this going to crash if missing? i.e. when deserializing any pre-existing file without History.